src/main/kotlin/no/fdk/dataset_catalog/rdf/RDFUtils.kt

```diff
@@ -81,26 +81,26 @@ fun Resource.safeAddDatasetListLinkedProperty(property: Property, value: List<Da
     }
 
 fun Resource.safeAddStringLiteral(property: Property, value: String?): Resource =
-    if(value == null) this
+    if (value == null && value != "") this
```

Comment:
mener du `if (value == null || value != "") this`?
value kan vel aldri være null og tom streng samtidig...